post-templates/embertimes-template.md

```diff
@@ -2,18 +2,17 @@
 title: The Ember Times - Issue No. XXS
 author:
   - the-crowd # replace with real authors from the author folder (add yourself if you're not there)
+date: 2021-XX-XXT00:00:00.000Z
```

Comment:
Is the date format ok? Always prefix `T00:00:00.000Z`?

Reply:
Yep. I think we can use https://raw.githubusercontent.com/ember-learn/ember-blog/master/content/the-ember-times-issue-170.md as a template.